python/openassetio/trait/managementPolicy.py

```diff
@@ -0,0 +1,71 @@
+#
+#   Copyright 2022 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+Traits returned by @ref hostApi.Manager.Manager.managementPolicy
+"Manager.managementPolicy"
+"""
+from typing import Union
+
+from ..Trait import Trait
+
+
+class ManagedTrait(Trait):
+    """
+    @ref hostApi.Manager.Manager.managementPolicy "managementPolicy"
+    trait specifying that a given @ref trait_set can be managed by the
+    @ref manager plugin.
+
+    There are three possible policies determined by applying/querying
+    this trait
+    * If a @fqref{TraitsData} "TraitsData" is not imbued with this
+      trait, then the Manager has no interest in participating in the
+      management of this kind of entity, or a supplied specification is
+      not publishable to a specific entity reference.
```

Comment:
[fix] Sorry I missed this the other day. I think we started talking about "Trait sets" here (which maybe didn't get updated in `constants.py`).

How about:

```
trait, then the Manager has no interest in participating in the
management of entities with the queried trait set, either for
read for for write.
```

Reply:
Updated in e475e96f